cmake/GoogleCloudCppCommon.cmake

```diff
@@ -62,6 +62,7 @@ else ()
         set(DOXYGEN_SEARCH_INCLUDES YES)
         set(DOXYGEN_INCLUDE_PATH
             "${PROJECT_SOURCE_DIR}" "${PROJECT_BINARY_DIR}"
+            "${PROJECT_BINARY_DIR}/google/cloud/examples"
```

Comment:
I believe the full path to the generated include file should be `${PROJECT_BINARY_DIR}/google/cloud/examples/hello_world_grpc/hello_world.grpc.pb.h` and it's included via `<google/cloud/examples/hello_world_grpc/hello_world.grpc.pb.h>`, so I don't understand why this include path would be needed. But if it fixes the build... LGTM.

Reply:
I think I can explain why. The `.grpc.pb.h` file reads:

```cc
#include "hello_world_grpc/hello_world.pb.h"
```

Why?  Because `protobuf` has strong opinions.  Could we fix that?  Maybe.  It would probably require invoking `protoc` from the top-level directory instead of the currrent dir.

Reply:
Let's certainly not call protoc ourselves. But thanks for explaining.